### ISSUE TITLE
Add User Authentication for ADE Jupyter Extensions outside Che

### ISSUE THREAD

**Issue by echyam** [MAAP-Project/ZenHub#64]
Many Jupyter extensions become unusuable outside of Che due to dependencies on data from the user profile.

**Comment by oddes** [comment]
@gchang 
Scope of this test is to validate login authentication for 
1. ESA/NASA existing user login validation
2. ESA/NASA new registered user validation

or 
we also need to validate all the existing functionality regression?

**Comment by gchang** [comment]
@oddes I'm not sure of the exact test case, but the test should verify that existing users, either NASA or ESA, can log in and access services.

**Comment by oddes** [comment]
Thanks @gchang  Login authentication works it generates token, as expected.
(1) Since there is no proper documentation to install all the extensions some cases dependencies I felt it is very hard as user point of view. It would be better to have a single package installer rather then installing each independent packages and dependencies [ i.e. dps_info, edsc_extension, ipycmc, submit_jobs].
(2) I have encountered an error while installing  "submit_jobs" so features related to it not working after I have logged in
(3) EDSC also not working for me since it needs service to connect to maap cmr, I am not sure we can do it.
